tracing-opentelemetry/README.md

```diff
@@ -59,24 +59,24 @@ The crate provides the following types:
 ### Basic Usage
 
 ```rust
-use opentelemetry::exporter::trace::stdout;
+use opentelemetry::sdk::export::trace::stdout;
 use tracing::{error, span};
-use tracing_subscriber::subscriber::SubscriberExt;
+use tracing_subscriber::layer::SubscriberExt;
 use tracing_subscriber::Registry;
 
 fn main() {
     // Install a new OpenTelemetry trace pipeline
-    let (tracer, _uninstall) = stdout::new_pipeline().install();
+    let tracer = stdout::new_pipeline().install_simple();
 
     // Create a tracing subscriber with the configured tracer
-    let telemetry = tracing_opentelemetry::subscriber().with_tracer(tracer);
+    let telemetry = tracing_opentelemetry::layer().with_tracer(tracer);
 
     // Use the tracing subscriber `Registry`, or any other subscriber
     // that impls `LookupSpan`
     let collector = Registry::default().with(telemetry);
 
     // Trace executed code
-    tracing::collect::with_default(collector, || {
+    tracing::subscriber::with_default(collector, || {
```

Comment:
this is changing the version on the master branch (v0.2.x) to use the released `tracing` version (v0.1.x)'s naming scheme. I don't think this is correct to do. Instead, we should base this PR against the `v0.1.x` branch.